span.go

```diff
@@ -9,6 +9,25 @@ import (
 	"github.com/opentracing/opentracing-go/ext"
 )
 
+// Span provides access to the essential details of the span, for use
+// by basictracer consumers.  These methods may only be called prior
+// to (*opentracing.Span).Finish().
+type Span interface {
+	opentracing.Span
+
+	// TraceID is a globally unique trace identifier
+	TraceID() int64
```

Comment:
Inasmuch as a `basictracer.Context` concept exists, I think I'd prefer that we just expose it here via `Context() Context`, copy bedamned. This is a nod to the fact that ("many moons ago") `Context` was pluggable and probably should be again in the future... i.e., initializing a `basictracer.Tracer` would require not just a `Recorder` but also some sort of context provider + context codec.

My two cents... (Though if we're going to expose traceid / spanid directly, I support your decision to leave the parentid and sampled status out of it for now)